src/formats/report.js

```diff
@@ -0,0 +1,46 @@
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const {TwingEnvironment, TwingLoaderFilesystem} = require( 'twing' );
+
+/**
+ * Internal dependencies
+ */
+const { getArgFromCLI } = require( '../utils/cli' );
+
+/**
+ * Get the source of a template to compile.
+ * 
+ * @param {string} subdir 
+ * @param {string} name 
+ * 
+ * @returns {string} Template contnet
+ */
+const getTemplateSrc = ( name ) => {
+	const templatePath = ( name ) => path.join( __dirname, `./templates/${name}.twig` );
+	const fileName = fs.existsSync( templatePath( name ) ) ? name : 'base';
+	
+	return fileName + '.twig';
+};
+
+let loader = new TwingLoaderFilesystem( path.join( __dirname, './templates' ) );
+let twing = new TwingEnvironment( loader, { debug: true } );
+
+module.exports = module.exports = function( reports ) {
+	
+	const reportName = getArgFromCLI('--report');
+	const reportDest = path.join( __dirname, `../../public/${reportName}.html` );
+	const reportTemplate = getTemplateSrc( reportName );
+	const context = {
+		reports,
+		title: `CSS Audit for ${reportName}`
+	};
+	
+	twing.render( reportTemplate, context ).then( ( output ) => {
```

Comment:
What do we get with `twing`, vs something like [lodash's `_.template`?](https://lodash.com/docs/4.17.15#template)

I don't really have much of a preference either way, but the promise callback means we can't return a direct success/error message from this function.

Reply:
Wow! I did not know this was part of lodash. I chose to Twing (a node implementation of Twig) b/c I find the template syntax to be very readable and thought others might be familiar with Twig since it is in the PHP world. Also since the template language is so fully featured, it seemed easier to get something going quickly and less barrier to entry for contributors since they can do so much in the templates.

Reply:
Works for me 👍